Sources/DataPipeline/DataPipeline.swift

```diff
@@ -3,6 +3,12 @@ import Segment
 
 public protocol DataPipelineInstance: CustomerIOInstance {
     var analytics: Analytics { get }
+    func processIdentifyFromBGQ(identifier: String, body: [String: Any]?)
+    func processScreenEventFromBGQ(identifier: String, name: String, timestamp: String?, properties: [String: Any])
+    func processEventFromBGQ(identifier: String, name: String, timestamp: String?, properties: [String: Any])
+    func processDeleteTokenFromBGQ(identifier: String, token: String)
+    func processRegisterDeviceFromBGQ(identifier: String, token: String, attributes: [String: Any]?)
+    func processPushMetricsFromBGQ(token: String, event: Metric, deliveryId: String, timestamp: String, metaData: [String: Any])
```

Comment:
we should have timestamps in here, commenting in the method i think we can get that value from.

Reply:
[Response](https://github.com/customerio/customerio-ios/pull/424#discussion_r1437854876)